src/assets/js/pages/EventPage/components/EventStatistics.js

```diff
@@ -44,6 +48,24 @@ export default class EventStatistics extends React.Component {
     return (
       <div className="event-statistics event-page__card">
         <h2>Applicant Demographics</h2>
+        <div className="row">
+          <h5>Gender Statistics</h5>
+          <VictoryPie
+            colorScale={["#8E44AD", "#43D2F0", "#AEF9D6", "#EF767A", "#7D7ABC" ]}
+            labelComponent={<VictoryTooltip />}
+            labelRadius={130}
+            labels={p => `${p.gender}: ${p.number}`} 
+            data={[
+              { gender: 'Male', number: statistics.genders['Male'] },
```

Comment:
We should dynamically load this data from what we get from the server - Instead of hardcoding 'Male', 'Female' etc in this array, we should iterate through the keys (or entries, your call) of the map to generate the `[{gender: 'xyz', number: 123}]` array needed here.